src/python_testing/TC_OCC_3_2.py

```diff
@@ -0,0 +1,243 @@
+#
+#    Copyright (c) 2024 Project CHIP (Matter) Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs: run1

```

Comment:
if you add CI test arguments, these have to run in CI.
This has to be added in https://github.com/project-chip/connectedhomeip/blob/master/.github/workflows/tests.yaml#L560

Reply:
I will add as soon as I implement namepiped CI calls.  This is addressed in https://github.com/project-chip/connectedhomeip/issues/34686